<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="JOBITO" failures="2" tests="2" name="Default test(failed)" time="0.081" errors="0" timestamp="2025-07-06T09:42:42 IST">
  <testcase classname="testNgDemo.TestNGDemoClass" name="registration" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNgDemo.TestNGDemoClass.registration(TestNGDemoClass.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- registration -->
  <testcase classname="testNgDemo.TestNGDemoClass" name="b" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNgDemo.TestNGDemoClass.b(TestNGDemoClass.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- b -->
  <testcase classname="testNgDemo.TestNGDemoClass" name="c" time="0.0">
    <ignored/>
  </testcase> <!-- c -->
  <testcase classname="testNgDemo.TestNGDemoClass" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
  <testcase classname="testNgDemo.TestNGDemoClass" name="d" time="0.0">
    <ignored/>
  </testcase> <!-- d -->
</testsuite> <!-- Default test(failed) -->
